parsl/monitoring/db_manager.py

```diff
@@ -380,10 +383,18 @@ def _migrate_logs_to_internal(self, logs_queue, queue_tag, kill_event):
                     self.pending_node_queue.put(x[-1])
 
     def _update(self, table, columns, messages):
-        self.db.update(table=table, columns=columns, messages=messages)
+        try:
+            self.db.update(table=table, columns=columns, messages=messages)
+        except Exception:
+            self.db.rollback()
```

Comment:
Can the rollback fail ?

Reply:
almost definitely because it involves talking to the database... that's probably something that should be fatal to monitoring though.

the log line should go right after the `except` statement so that it is clearer what is happening

Reply:
Added another `try except` around rollback